server/middleware/setUpAuthentication.ts

```diff
@@ -42,10 +43,14 @@ export default function setupAuthentication() {
   router.use(passport.session())
   router.use(flash())
 
-  router.get('/autherror', (_req, res) => {
-    res.status(401)
-    return res.render('autherror')
-  })
+  router.get(
+    '/autherror',
+    dpsComponents.getPageComponents({ dpsUrl: config.serviceUrls.digitalPrison }),
```

Comment:
We can do this to ensure that the autherror page still gets a header and footer!